drivers/sensor/icm42688/icm42688.h

```diff
@@ -294,7 +294,7 @@ static inline void icm42688_accel_ms(struct icm42688_cfg *cfg, int32_t in, int32
 	*out_ms = in_ms / (sensitivity * 1000000LL);
 
 	/* micrometers/s^2 */
-	*out_ums = ((llabs(in_ms) - (llabs(*out_ms) * sensitivity * 1000000LL)) / sensitivity);
+	*out_ums = (in_ms - (*out_ms * sensitivity * 1000000LL)) / sensitivity;
```

Comment:
Is this change related to `test: verify accel/gyro data for icm42688`?

Reply:
Yes, it was incorrect to take the diff of the abs